### PR TITLE
Add Skapps to health check by setting the max http header size

### PR DESCRIPTION
**Pull request by MSevey**
This MR sets the `--max-http-header-size` to the same as the handshake integration and then adds the uniswap and develop momentum skylinks to the health checks.

### CODE REVIEW

File: packages/health-check/src/verboseChecks.js [hunk 1]
```diff
@@ -551,6 +370,59 @@ function uniswapIndexFileCheck(done) {
...
+
+// uniswapHNSResolverRedirectCheck returns the result of trying to download the skylink
+// for the Uniswap Application via the HNS resolver endpoint without the
+// trailing slash
+function uniswapHNSResolverRedirectCheck(done) {
+  const linkInfo = {
+    description: "Uniswap HNS Resolver Redirect",
+    skylink: "hnsres/doesn",
+    bodyHash: "44a3f0f56861ae841a6cb19cb0b3edf98ad610f8",
+    metadata: null,
```
Comment (MSevey) [inline packages/health-check/src/verboseChecks.js:420]: @kwypchlo these don't have skyfile metadata defined so the response is `undefined`. Will setting this to `null` work with the `lodash.IsEqual`?

Comment (kwypchlo) [reply packages/health-check/src/verboseChecks.js:420]: Nope, but you might want to set `metadata: undefined` and it should work in theory :)

Comment (kwypchlo) [reply packages/health-check/src/verboseChecks.js:420]: Why doesn't it have the metadata though? Is this correct?

File: packages/health-check/src/metadata.js [hunk 2]
```diff
@@ -0,0 +1,1043 @@
+const developMomentumMetadata = {
```
Comment (kwypchlo) [inline packages/health-check/src/metadata.js:1]: Can you convert this file to multiple `.json` files and put them in `fixtures` directory?

File: packages/health-check/Dockerfile [hunk 3]
```diff
@@ -9,3 +9,4 @@ COPY src/* src/
 EXPOSE 3100
 ENV NODE_ENV production
 CMD [ "node", "src/index.js" ]
+CMD [ "node", "--max-http-header-size=64000", "src/index.js" ]
```
Comment (kwypchlo) [inline packages/health-check/Dockerfile:12]: remove the old `CMD` from the previous line

**Review by kwypchlo — CHANGES_REQUESTED** [review]

**Comment by MSevey** [comment]
update `/doesn` url to `uniswap-dex`

File: packages/health-check/src/verboseChecks.js [hunk 4]
```diff
@@ -69,13 +70,42 @@ function dappExampleCheck(done) {
   skylinkVerification(done, linkInfo);
 }
 
+const developMomentumBodyHash = "53b44a9d3cfa9b3d66ce5c29976f4383725d3652";
+const developMomentumMetadata = JSON.parse(fs.readFileSync("./fixtures/developMomentumMetadata.json"));
```
Comment (kwypchlo) [inline packages/health-check/src/verboseChecks.js:74]:
```suggestion
const developMomentumMetadata = require("./fixtures/developMomentumMetadata.json");
```

File: packages/health-check/src/verboseChecks.js [hunk 5]
```diff
@@ -228,134 +258,16 @@ function skyBinRedirectCheck(done) {
   skylinkVerification(done, linkInfo);
 }
 
+const skyGalleryBodyHash = "077e54054748d278114f1870f8045a162eb73641";
+const skyGalleryMetadata = JSON.parse(fs.readFileSync("./fixtures/skygalleryMetadata.json"));
```
Comment (kwypchlo) [inline packages/health-check/src/verboseChecks.js:262]:
```suggestion
const skyGalleryMetadata = require("./fixtures/skygalleryMetadata.json");
```

**Review by kwypchlo — CHANGES_REQUESTED** [review]

File: packages/health-check/src/verboseChecks.js (hunk 4 above)
Comment (MSevey) [reply packages/health-check/src/verboseChecks.js:74]: well isn't that fancy. I assumed it was going to be something neat like this but I didn't know what to google haha.

**Review by kwypchlo — APPROVED** [review]

**Review by kwypchlo — APPROVED** [review]

**Pull request merged**